src/main/scala/Core/main.scala

```diff
@@ -10,16 +10,40 @@ package Core
 import chisel3._
 
 // Remove this eventually...
-import flexpret.core.Core
+import flexpret.core.{VerilatorTop, FpgaTop}
 import flexpret.core.FlexpretConfiguration
 
 object CoreMain {
-  def main(args: Array[String]): Unit = {
+  
+  def verilatorMain(args: Array[String]) : Unit = {
     val confString = args(0)
     val chiselArgs = args.slice(1, args.length)
     val coreConfig = FlexpretConfiguration.parseString(confString)
 
     // Pass configuration to FlexPRET processor.
-    (new chisel3.stage.ChiselStage).emitVerilog(new Core(coreConfig), chiselArgs)
-  }
+    (new chisel3.stage.ChiselStage).emitVerilog(new VerilatorTop(coreConfig), chiselArgs)
+  } 
+  
+  def fpgaMain(args: Array[String]) : Unit = {
+    val confString = args(0)
+    val chiselArgs = args.slice(1, args.length)
+    val coreConfig = FlexpretConfiguration.parseString(confString)
+
+    // Pass configuration to FlexPRET processor.
+    (new chisel3.stage.ChiselStage).emitVerilog(new FpgaTop(coreConfig), chiselArgs)
+  } 
+    
+  def main(args: Array[String]): Unit = {
+    if (args.length < 2) {
+      sys.error(s"Usage sbt run [target] [configString]") 
+    }
+    val target = args(0)
+    if (target == "verilator") {
+      verilatorMain(args.slice(1, args.length))
+    } else if (target == "fpga"){
+      fpgaMain(args.slice(1, args.length))
+    } else {
+      sys.error(s"Unrecognized target $target. Currently only supports `emulator`")
```

Comment:
"Currently only supports \`emulator\`"  - should this say "Currently only supports \`verilator\` or \`fpga\`"?

Reply:
Yes, thanks